### PR TITLE
Make ratchet tree section clearer.

### PR DESCRIPTION
**Pull request by Bren2010**

### CODE REVIEW

**Comment by seanturner** [comment]
202105 Interim: To be reviewed by editors.

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -828,28 +828,28 @@ local tree state as described in the prior section, the generator must broadcast
 this update to other members of the group in a Commit message, who
 apply it to keep their local views of the tree in
 sync with the sender's.  More specifically, when a member commits a change to
-the tree (e.g., to add or remove a member), it transmits an UpdatePath message
-containing a set of public and encrypted private
-values for intermediate nodes in the direct path of a leaf. The
+the tree (e.g., to add or remove a member), it transmits an UpdatePath
+containing a set of public keys and encrypted private
+keys for intermediate nodes in the direct path of its leaf. The
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md:833]:
```suggestion
containing a set of public keys and encrypted path secrets
for intermediate nodes in the direct path of its leaf. The
```

**Review by bifurcation — CHANGES_REQUESTED** [review]
One minor fix, otherwise ready to go.

**Review by raphaelrobert — APPROVED** [review]

File: draft-ietf-mls-protocol.md (hunk 1 above)
Comment (seanturner) [reply draft-ietf-mls-protocol.md:833]: minor correctness fix.

**Review by bifurcation — APPROVED** [review]

**Pull request merged**